tests/macros/test_hive.py

```diff
@@ -19,7 +19,7 @@
 import unittest
 from datetime import datetime
 
-from airflow.macros import hive
+from airflow.providers.apache.hive.macros import hive
```

Comment:
Tests also should co-locate with other tests of this provider not under and macros package